build/optaplanner-build-parent/pom.xml

```diff
@@ -21,7 +21,7 @@
     <!-- ************************************************************************ -->
 
     <!-- SNAPSHOT and KIE dependency versions -->
-    <version.org.kie.kogito>2.0.0-SNAPSHOT</version.org.kie.kogito>
+    <version.org.kie.kogito>1.5.0-SNAPSHOT</version.org.kie.kogito>
     <!-- Keep in sync with version.org.kie7 in kogito-runtimes: https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-build-parent/pom.xml#L185 -->
     <version.org.kie7>7.52.0.Beta1</version.org.kie7>
```

Comment:
No upgrade to Drools 7.52.0.Final?
(It may be perfectly fine if Kogito decided not to make that upgrade as well, although they usually do.)

Reply:
The same version is in kogito-runtime 1.5.x and master branches.